Makefile.am

```diff
@@ -16,6 +16,9 @@ dist_systemdunit_DATA = \
 systemdunit_DATA = \
 	dev-disk-by\\x2dlabel-eos\\x2dswap.swap \
 	var-endless\\x2dextra.mount
+
+udevrulesdir=/lib/udev/rules.d
```

Comment:
I don't think you need to, this is already behind a configure arg which is not enabled for distcheck.